Zend/zend_gc.c

```diff
@@ -1282,7 +1282,7 @@ static void gc_scan(zend_refcounted *ref, gc_stack *stack)
 					GC_REF_SET_COLOR(ht, GC_WHITE);
 					GC_STACK_PUSH((zend_refcounted *) ht);
 					for (; n != 0; n--) {
-						if (Z_REFCOUNTED_P(zv)) {
+						if (Z_OPT_COLLECTABLE_P(zv)) {
```

Comment:
Don't mix OPT/no-OPT changes in this patch. Use Z_COLLECTABLE here.

Reply:
Oops, that's a copy&paste error. Thanks!